### PR TITLE
Feature/add bem

### PR DESCRIPTION
**Pull request by MrSkorohod**
The folder structure has been changed, BEM classes have been added.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+    <meta charset="UTF-8">
+    <title>Productly</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="https://meyerweb.com/eric/tools/css/reset/reset.css" />
+    <link rel="stylesheet" href="main.css" />
+</head>
+<body>
+    <header class="header container">
+        <div class="logotype">
+            <img src="img/logo.svg" alt="Logo picture">
```
Comment (ElizavetaSelivanova) [inline index.html:13]: Self-closing tag

File: index.html [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+                <li><a href="#">Pricng</a></li>
+                <li><a href="#">Resources</a></li>
+            </ul>
+        </nav>
+        <div class="header-buttons">
+            <button class="gray-button">Sign In</button>
+            <button class="yellow-button">Sign Up</button>
+            <div class="hamburger">
+                <a href="#">
+                    <svg viewBox="0 0 100 80" width="40" height="40">
```
Comment (ElizavetaSelivanova) [inline index.html:29]: Move svg to a separated file

File: index.html [hunk 3]
```diff
@@ -0,0 +1,200 @@
...
+                        <span>By <span class="author-name">Abdullah</span> |  03 March 2019</span>
+                        <h3 class="active-title">Motivation Is The First Step To Success</h3>
+                    </div>
+                    <div>
+                        <img src="img/card_image_3.jpg" alt="Card picture">
+                        <span>By <span class="author-name">Abdullah</span> |  03 March 2019</span>
+                        <h3>Success Steps For Your Personal Or Business Life</h3>
+                    </div>
+                </div>
+                <div class="carousel-buttons">
```
Comment (ElizavetaSelivanova) [inline index.html:157]: You can use <ul> for this list of dots

File: index.html [hunk 4]
```diff
@@ -0,0 +1,200 @@
...
+            </nav>
+        </div>
+        <div class="tel-form">
+            <h3>knowing you're always on the best energy deal.</h3>
+            <input type="tel" placeholder="Enter your phone Number">
+            <button class="yellow-button">Sign up Now</button>
+        </div>
+    </footer>
+</body>
+</html>
```
Comment (ElizavetaSelivanova) [inline index.html:200]: Add an empty line at the end of file

File: stylesheets/abstracts/_variables.scss [hunk 5]
```diff
@@ -0,0 +1,79 @@
+// -----------------------------------------------------------------------------
+// This file contains all application-wide Sass variables.
+// -----------------------------------------------------------------------------
+
+
```
Comment (ElizavetaSelivanova) [inline stylesheets/abstracts/_variables.scss:5]: Remove unnecessary indents, max - 1 line indent

File: stylesheets/base/_typography.scss [hunk 6]
```diff
@@ -0,0 +1,118 @@
...
+ */
+
+body {
+  color: $text-color;
+  font: 400 14px / 28px $text-font;
+}
+.header-navigation {
+  color: $text-color;
+}
+.logotype h1{
```
Comment (ElizavetaSelivanova) [inline stylesheets/base/_typography.scss:12]: Add space before `{`

File: stylesheets/base/_typography.scss [hunk 7]
```diff
@@ -0,0 +1,118 @@
...
+  color: $logo-color;
+  font: 700 30px / 34px $text-font;
+}
+.yellow-button{
```
Comment (ElizavetaSelivanova) [inline stylesheets/base/_typography.scss:16]: Try to specify common styles for buttons

**Review by ElizavetaSelivanova — COMMENTED** [review]
Good job!

Try to review your code before sending PRs, try to formatting your code

File: index.html (hunk 3 above)
Comment (irina-chernik) [reply index.html:157]: and you can't put block element inside inline)

File: stylesheets/base/_helpers.scss [hunk 8]
```diff
@@ -0,0 +1,55 @@
+// -----------------------------------------------------------------------------
+// This file contains CSS helper classes.
+// -----------------------------------------------------------------------------
+
+/**
+ * Clear inner floats
+ */
+.clearfix::after {
```
Comment (irina-chernik) [inline stylesheets/base/_helpers.scss:8]: do you need this?

File: stylesheets/base/_typography.scss [hunk 9]
```diff
@@ -0,0 +1,118 @@
+/**
+ * Basic typography style for copy text
+ */
+
+body {
+  color: $text-color;
+  font: 400 14px / 28px $text-font;
+}
```
Comment (irina-chernik) [inline stylesheets/base/_typography.scss:8]: add empty line between selector blocks

File: stylesheets/base/_typography.scss [hunk 10]
```diff
@@ -0,0 +1,118 @@
...
+  h2 {
+    color: $brand-color;
+    font: 700 36px / 54px $text-font;
+  }
+  p {
+    color: $paragraph-color;
+    font: 500 16px / 26px $text-font;
+  }
+}
+.section-news {
```
Comment (irina-chernik) [inline stylesheets/base/_typography.scss:79]: typography file is usually used for styling common text element (mostly titles, paragraphs, quotes)
all other section, buttons font styles should be where all other styles for this elements specified
here is an example of what is usually included in _typography.scss
https://gist.github.com/sebastiano-guerriero/2d05b319b9ae7618d99809797a67c0a0#file-typography_10-scss

File: stylesheets/layout/_section-news.scss [hunk 11]
```diff
@@ -0,0 +1,41 @@
+.section-news {
+  padding-bottom: 120px;
+  border-bottom: 1px solid $light-grey  ;
```
Comment (irina-chernik) [inline stylesheets/layout/_section-news.scss:3]: remove spaces at the end

File: index.html [hunk 12]
```diff
@@ -0,0 +1,200 @@
...
+                </a>
+            </div>
+        </div>
+
+    </header>
+    <main>
+        <article>
+            <section class="main-section container">
+                <div class="main-section-text">
+                    <h1 class="main-headers">The Design Thinking superpowers</h1>
```
Comment (irina-chernik) [inline index.html:43]: class main-header probably, I see here only one

**Comment by angelinabylitskaya** [comment]
Closed due to wrong branching strategy.

**Pull request closed without merging**